src/rgw/rgw_log.cc

```diff
@@ -128,14 +128,13 @@ class UsageLogger {
     round_timestamp = ts.round_to_hour();
   }
 
-  void insert_user(utime_t& timestamp, const rgw_user& user, rgw_usage_log_entry& entry) {
+  void insert_user(utime_t& timestamp, const string& user, rgw_usage_log_entry& entry) {
```

Comment:
Oh, I missed that, sorry. However, in such case you might have a conflict with multi-tenancy because carrying the `user_name:subuser_name` through `rgw_user`-typed fields in `rgw_usage_log_entry` (`UsageLogger::insert` takes it).